testing/test_session.py

```diff
@@ -253,3 +256,34 @@ def pytest_sessionfinish():
     """)
     res = testdir.runpytest("--collect-only")
     assert res.ret == EXIT_NOTESTSCOLLECTED
+
+
+@pytest.mark.parametrize("path", ["root", "{relative}/root", "{environment}/root"])
+def test_rootdir_option_arg(testdir, path):
+    if 'relative' in path:
+        path = path.format(relative=os.getcwd())
+    if 'environment' in path:
+        os.environ['PY_ROOTDIR_PATH'] = os.getcwd()
```

Comment:
Better to use `monkeypatch.setenv` instead so this variable is removed at the end of the test.